config/apply-certs.sh

```diff
@@ -8,17 +8,29 @@ DIR="$(pwd)"
 NAMESPACE="dss-main"
 CLIENTS_CERTS_DIR="$DIR/generated/$NAMESPACE/client_certs_dir"
 NODE_CERTS_DIR="$DIR/generated/$NAMESPACE/node_certs_dir"
+CA_KEY_DIR="$DIR/generated/$NAMESPACE/ca_key_dir"
+CA_CRT_DIR="$DIR/generated/$NAMESPACE/ca_certs_dir"
 JWT_PUBLIC_CERTS_DIR="$DIR/jwt-public-certs"
+UPLOAD_CA_KEY=false
 
 # Delete previous secrets in case they have changed.
 kubectl create namespace "$NAMESPACE" || true
 
 kubectl delete secret cockroachdb.client.root || true
 kubectl delete secret cockroachdb.client.root --namespace "$NAMESPACE" || true
 kubectl delete secret cockroachdb.node --namespace "$NAMESPACE" || true
+kubectl delete secret cockroachdb.node --namespace "$NAMESPACE" || true
```

Comment:
Looks like this line is a duplicate of the previous one?

Reply:
yup, done